sdk/tables/Azure.Data.Tables/README.md

```diff
@@ -257,13 +257,13 @@ For more information see the [Code of Conduct FAQ][coc_faq] or contact
 [opencode@microsoft.com][coc_contact] with any additional questions or comments.
 
 <!-- LINKS -->
-[tables_rest]: https://docs.microsoft.com/en-us/rest/api/storageservices/table-service-rest-api
+[tables_rest]: https://docs.microsoft.com/rest/api/storageservices/table-service-rest-api
 [azure_cli]: https://docs.microsoft.com/cli/azure
 [azure_sub]: https://azure.microsoft.com/free/
 [table_client_nuget_package]: https://www.nuget.org/packages?q=Azure.Data.Tables
 [table_client_samples]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/samples
 [table_client_src]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/src
-[api_reference]: https://docs.microsoft.com/en-us/azure/cosmos-db/table-introduction
+[api_reference]: https://docs.microsoft.com/azure/cosmos-db/table-introduction
 [logging]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/Diagnostics.md
 [contrib]: ./CONTRIBUTING.md
```

Comment:
IIRC, relative links here were potentially breaking in some scenarios, but that may have been mostly related to samples when published.

Reply:
This isn't used actually, so I'll just remove it